js/about/preferences.js

```diff
@@ -300,9 +301,13 @@ class BitcoinDashboard extends ImmutableComponent {
   }
   get qrcodeOverlayFooter () {
     return <div>
-      <div id='coinbaseLogo' />
-      <div id='appstoreLogo' />
-      <div id='playstoreLogo' />
+      {
+        coinbaseCountries.indexOf(this.props.ledgerData.get('countryCode')) > -1
+        ? <div id='coinbaseLogo' />
+        : null
+      }
+      <a href='https://itunes.apple.com/us/app/coinbase-bitcoin-wallet/id886427730?mt=8' target='_blank' id='appstoreLogo' />
+      <a href='https://play.google.com/store/apps/details?id=com.coinbase.android' target='_blank' id='playstoreLogo' />
```

Comment:
why are we linking to the coinbase app for countries where coinbase is not supported?